### PR TITLE
add couch.requests metric with bucketed timings

### PR DESCRIPTION
**Pull request by snopoke**
https://dimagi-dev.atlassian.net/browse/SAAS-12083

Add `couch.requests` counter metric with duration added as a bucketed tag.

### CODE REVIEW

File: couch/parsers.py [hunk 1]
```diff
@@ -24,14 +24,33 @@ def parse_couch_logs(logger, line):
         logger.exception('Failed to parse log line')
         return None
 
-    return ('couch.timings', timestamp, request_seconds, {
+    return [
+        get_couch_timing_gauge(timestamp, url, database, http_method, status_code, couch_url, request_seconds)
```
Comment (snopoke) [inline couch/parsers.py:28]: I kept this metric for now but we can remove it if it's no longer needed

File: couch/parsers.py [hunk 2]
```diff
@@ -24,14 +24,33 @@ def parse_couch_logs(logger, line):
         logger.exception('Failed to parse log line')
         return None
 
-    return ('couch.timings', timestamp, request_seconds, {
+    return [
+        get_couch_timing_gauge(timestamp, url, database, http_method, status_code, couch_url, request_seconds),
+        get_couch_requests_counter(timestamp, url, database, http_method, status_code, couch_url, request_seconds)
+    ]
```
Comment (dannyroberts) [inline couch/parsers.py:30]: Nice I didn't realize you can return a list here. (Maybe we do that elsewhere and I just forgot.)

**Review by dannyroberts — APPROVED** [review]
Looks great!

File: couch/parsers.py (hunk 2 above)
Comment (snopoke) [reply couch/parsers.py:30]: we also do this with the nginx metrics

**Pull request merged**